docs/pages/2020_BIGMAP/index.rst

```diff
@@ -20,14 +20,33 @@ BIG-MAP meeting AiiDA tutorial
 
 This is the content of the AiiDA tutorial organised for the BIG-MAP meeting on 2 December 2020.
 
+
+Setup
+-----
+
+The tutorial will be run on the |AiiDAlab tutorials cluster|, just click the link and log in with a username and password of your choosing.
+
+.. important::
+
+  Make a note of your password!
+  If you lose it, you can make a new account, but you will lose your progress.
+
+That's it!
+You're now ready to start with the tutorial.
```

Comment:
```suggestion
It will take a few minutes for your server to start up on first login, after that you are all set and ready to start with the tutorial!

```